input/fsh/Profiles clinical datapoints/DisutilityOfCare.fsh

```diff
@@ -1,35 +1,41 @@
+Profile: ReoperationSurgery
 Parent: Procedure 
-Id: involved-margins-reoperation
-Title: "Involved Margins Reoperation"
-Description: "A reoperation due to involved margins after primary surgery"
+Id: reoperation-surgery
+Title: "Re-operation surgery"
+Description: "Represents whether the patient has had a re-operation since their surgery for breast cancer and whether it was due to involved margins."
 * insert PublicationProfileRuleset
-* code from InvolvedMarginsReoperationTypeVS (required)
+* category = SCT#261554009 "Reoperation"
+* code from ReoperationTypeVS (required)
 * subject only Reference(BreastCancerPatient)
+* partOf MS
+  * ^short = "Original procedure that prompted the reoperation"
+  * ^definition = "A larger event of which this particular procedure is a component or step. In this case, the original procedure that prompted the reoperation."
 * performed[x] only dateTime 
-* performedDateTime and code and subject MS
+* reasonCode from ReoperationReasonVS (preferred)
 * reasonReference only Reference (PrimaryBreastCancerCondition)
-* extension contains ProcedureCausedByEx named causedBy 0..* MS
-* extension[causedBy] ^short = "Original procedure that prompted the reoperation"
-* extension[causedBy].value[x] only Reference(Procedure)
+* category and code and subject and performedDateTime and reasonCode and reasonReference MS
 
-Instance: ReoperationPatient147
-InstanceOf: InvolvedMarginsReoperation
+Instance: ReoperationSurgeryPatient147
+InstanceOf: ReoperationSurgery
 Description: "Example of a reoperation due to involved margins after primary surgery"
 * status = ProcedureStatusCS#completed 
-* code = IchomReoperation#4 "Mastectomy with immediate reconstruction"
+* category = SCT#261554009 "Reoperation"
+* code = BreastSurgeryTypesCodeSystem#2 "Mastectomy without immediate reconstruction"
 * subject = Reference(BreastCancerPatient147)
 * performedDateTime = "2022-07-09"
+* reasonCode = SCT#1156344002 "Presence of primary malignant neoplasm of breast at surgical margin in excised specimen of breast"
 * reasonReference = Reference(PrimaryBreastCancerPatient147)
+* partOf = Reference(BreastCancerSurgeryPatient147)
 
-Mapping: InvolvedMarginsReoperationToICHOM
-Source:	InvolvedMarginsReoperation
+Mapping: ReoperationSurgeryToICHOM
+Source:	ReoperationSurgery
 Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
-Id: reoperationmapping
-Title: "Involved margins reoperation to ICHOM set"
-Description: "Mapping of the involved margins reoperation procedure to the ICHOM breast cancer PCOM set." 	
-* -> "Involved margins reoperation"
-* performed[x] -> "Positive margins reoperation date"
+Id: reoperation-surgery-mapping
+Title: "Reoperation surgery to ICHOM set"
+Description: "Mapping of the reoperation surgery to the ICHOM breast cancer PCOM set" 	
+* -> "Surgery"
+* reasonCode -> "Involved margins reoperation"
+* performed[x] -> "Positive margins reoperation date or surgery date"
```

Comment:
```suggestion
* performed[x] -> "Surgery date"
* performed[x] -> "Positive margins reoperation date"
```

Reply:
It can be both actually, so it can be the positive margins reopration or just the surgery date of the reoperation